_includes/v23.1/ui/insights.md

```diff
@@ -26,18 +26,23 @@ Transaction executions with the **High Contention** insight are transactions tha
 {% if page.cloud != true -%}
 The following screenshot shows the execution of a transaction flagged with **High Contention**:
 
-<img src="{{ 'images/v22.2/transaction_execution.png' | relative_url }}" alt="Transaction execution" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v23.1/transaction_execution.png' | relative_url }}" alt="Transaction execution" style="border:1px solid #eee;max-width:100%" />
 {% endif -%}
 
 To view [details of the execution](#transaction-execution-details), click an execution ID in the **Latest Transaction Execution ID** column.
 
 - **Latest Transaction Execution ID**: The execution ID of the latest execution with the transaction fingerprint.
 - **Transaction Fingerprint ID**: The transaction fingerprint ID of the latest transaction execution.
 - **Transaction Execution**: The transaction fingerprint of the latest transaction execution.
+- **Status**: The transaction status.
```

Comment:
The status of the transaction: Failed or Completed.

Took the same format from the following doc on Sessions: https://www.cockroachlabs.com/docs/stable/ui-sessions-page.html#sessions-table